lib/transformer/caster/exporter.rb

```diff
@@ -2,10 +2,19 @@ module Transformer
   module Caster
     class Exporter < Base
       def analyze
-        @dataset_cast.graph_methods.each_pair do |key, value|
-          next unless value
+        GraphMethods.all.each do |key, attribute|
+          value = @dataset_cast.graph_methods[key]
+          query = Atlas::Gquery.all.detect { |gq| gq.key == key }
```

Comment:
`Atlas::Gquery.find(key)`

---

I'm not a big fan of using `Gquery` for this; `Inputs` are for writing in ETEngine, `Gqueries` are for reading in ETEngine. These gqueries would not be executable in ETEngine.

These feel like something different... something entirely new. I think we should find a new name/class for GQL in ETLocal. `ETLocalInput`, or `PreProcessorInput`, or `NotAnInitializerInput`. 😄 (I don't like any of those names TBH, perhaps you can think of something better?)

This might allow attaching other useful metadata (such as describing the object and attribute which it affects, rather than having `graph_methods` as part of node/edge documents?

```
- updates = industry_useful_demand_for_chemical_aggregated_industry
- attribute = demand
- query = 2.0 * ETLOCAL(chemical_demand)
```